install_config/router/default_haproxy_router.adoc

```diff
@@ -1498,18 +1498,45 @@ To verify the actual amount of ARP entries for IPv4, run the following:
 ----
 
 If the number begins to approach the `net.ipv4.neigh.default.gc_thresh3`
-threshold, increase the values. Run the following on the nodes running the
-router pods. The following sysctl values are suggested for clusters with large
-numbers of routes:
+threshold, increase the values. Get the current value by running:
 
 ----
-net.ipv4.neigh.default.gc_thresh1 = 8192
-net.ipv4.neigh.default.gc_thresh2 = 32768
-net.ipv4.neigh.default.gc_thresh3 = 65536
+# sysctl net.ipv4.neigh.default.gc_thresh1
+net.ipv4.neigh.default.gc_thresh1 = 128
+# sysctl net.ipv4.neigh.default.gc_thresh2
+net.ipv4.neigh.default.gc_thresh2 = 512
+# sysctl net.ipv4.neigh.default.gc_thresh3
+net.ipv4.neigh.default.gc_thresh3 = 1024
 ----
 
-To make these settings permanent across reboots, create a
-link:https://access.redhat.com/solutions/1305833[custom tuned profile].
+The following sysctl sets the variables to the {product-title} current default values.
+
+----
+# sysctl net.ipv4.neigh.default.gc_thresh1=8192
+# sysctl net.ipv4.neigh.default.gc_thresh2=32768
+# sysctl net.ipv4.neigh.default.gc_thresh3=65536
+----
+
+To make these settings permanent across reboots, until an {product-title} upgrade 
+occurs, edit the following file on each node:
+
+----
+/usr/lib/tuned/atomic-openshift-node-host/tuned.conf
```

Comment:
https://access.redhat.com/solutions/1305833

This article suggests creating a new profile in /etc which includes the original and then sets just the things it wants set. Updates to the original will get applied, but customization the user makes will never be overwritten (which is what we want)

Any suggestion that a user edit a file in `/usr` is wrong.